resource-server-webapp/pom.xml

```diff
@@ -158,7 +170,7 @@
 				<executions>
 					<execution>
 						<id>sign-artifacts</id>
-						<phase>verify</phase>
+						<phase>deploy</phase>
```

Comment:
Is this related to the jaxb/logback changes?
Is this needed for Java 11?

Reply:
If I'm remembering correctly, with 'verify', the build was failing with Java 11.  I changed it to 'deploy' to resolve because I figured we don't need to sign artifacts anyways unless we're doing a deploy. Now that I think about it, if we run a deploy the same error will probably still occur :(

Reply:
Okay so I retested using 'verify' and I get build failure with both Java 8 and Java 11.  Here's the error, anyone have insights:
```[DEBUG] Generating signature for /home/groybal/projects/unicon/uportal/forks/resource-server/resource-server-webapp/target/ResourceServingWebapp.war
[DEBUG] 2.2.19
[GNUPG:] INV_SGNR 9
[GNUPG:] FAILURE sign 17
gpg: no default secret key: No secret key
gpg: signing failed: No secret key```

Reply:
I'd check

```bash
gpg --list-keys
```
to see if there is a key already setup on your system

If there isn't try running

```bash
gpg --generate-key
```

Reply:
Yep, that fixed it (generating key).  Still should use 'deploy' I think but let me know if I should change back to 'verify'.

Reply:
I'm good with it staying in deploy as well.
Though open to differing opinions.